libqtile/core/manager.py

```diff
@@ -152,16 +154,28 @@ def __init__(
                 self.groups.append(sp)
                 self.groups_map[sp.name] = sp
 
-        self._eventloop = eventloop
-        self.setup_eventloop()
-        self.server = IPCCommandServer(self)
+        # It fixes problems with focus when clicking windows of some specific clients like xterm
+        # 2020-12-31: unsure if this still fixes anything
+        def noop(qtile):
+            pass
+        self.config.mouse += (Click([], "Button1", lazy.function(noop), focus="after"),)
 
-        self.current_screen = None
-        self.screens = []
+    def dump_state(self, buf):
+        try:
+            pickle.dump(QtileState(self), buf, protocol=0)
+        except:  # noqa: E722
+            logger.exception('Unable to pickle qtile state')
```

Comment:
Can we catch this `Exception` and put it in the log message?

Reply:
That's exactly what this code already does.